.github/workflows/jwt-cpp.yml

```diff
@@ -44,10 +44,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ref: [ 0.6.0 ]
+        ref: [ 0.7.0 ]
```

Comment:
Why are we not testing `0.6.0` anymore?

Reply:
0.6.0 does not compile on the latest compiler because of a build error.

Reply:
We should fix that error. What is the error?

Reply:
https://github.com/wolfSSL/wolfssl/actions/runs/11485310894 for example passes.

Reply:
Note that the example you gave uses the `ubuntu-22.04` runner. It will fail on `ubuntu-latest`. See: https://github.com/wolfSSL/wolfssl/actions/runs/11350595837/job/31569431321.

Reply:
If you don't want to invest time investigating the source of the error, we should at least run those tests on the version of Ubuntu it passes on. You can add the runner choice to the strategy to run 0.6 on the lts version and 0.7 on latest.

Reply:
It's not that I don't want to investigate the source of the error. It's simply adding in `#include <cstdint.h>` into `base.h`, which is what `jwt-cpp` has [already been done](https://github.com/Thalhammer/jwt-cpp/compare/v0.6.0...v0.7.0#diff-7884d932b55bcb4b9bbf516c2f3f7e55624c0dfdf8656783ec42c8d3d694db5eR6). I have done as you suggested and just added a `matrix`.